### DISCUSSION TITLE
List money borrowed

### DISCUSSION THREAD

**Discussion by 0smic** [Ivy-Apps/ivy-wallet#3366] [Ideas]
Hello,
I hope u guys create a feature, that we can list how much money we are given to whom and etc.
It can't be added as an expense, cause we get that back, also can't be added in account transfer cause we give money to lots of friends. So the list of ac grows, it would be messy or smtg.  Hope u get it. 
And also add emi and loans features to keep track of it.

**Comment by ILIYANGERMANOV** [comment]
Have you tried our "Loans" feature? It sounds like it
